app/__init__.py

```diff
@@ -34,6 +34,7 @@ def create_app(testing=False):
         register_routes(app)
 
     with app.app_context():
+        db.create_all()
```

Comment:
eventually the tables would already exist right? or does this create them if they don't already?

Reply:
If I start and stop the app they do not get reinitialized, but I also had the same question. I was unsure needed to be wrapped in a conditional, though from what I see in examples online, it does not.

ex. https://stackoverflow.com/questions/20744277/sqlalchemy-create-all-does-not-create-tables

Reply:
Funny. All I had to do was look at the official docs... 😸 
https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/quickstart/#create-the-tables